index.js

```diff
@@ -12,12 +12,15 @@ const morgan = require('morgan'),
     request = require('request'),
     favicon = require('serve-favicon'),
     path = require('path'),
+    compression = require('compression'),
     apiRouter = require('./server/apiRouter'),
     publicRoutes = require('./server/routes/public'),
     cloudinary = require('cloudinary'),
     authenticatedRoutes = require('./server/routes/authenticated'),
     auth = require('./server/controllers/auth'),
+    bunyan = require('bunyan'),
```

Comment:
Can't see exactly where the bunyan module is being used in the index.js file other than it being required.

Reply:
Yes, you are right my intention was to use it for logging since using the console to log stuff is not advisable because of its synchronous blocking nature. I will remove it for now until the need arises

Reply:
Done